cmd/dapr.go

```diff
@@ -46,6 +46,12 @@ type daprVersion struct {
 	RuntimeVersion string `json:"Runtime version"`
 }
 
+type osType string
+
+const (
+	WINDOWS osType = "windows"
+)
```

Comment:
nit: move it to a common place?

```suggestion
type osType string

const (
	WindowsOsType osType = "windows"
)
```

If it's only used for `runtime.GOOS`, we can also skip this change and directly use `"windows"` string.

Reply:
it is being used in `runtime.GOOS` only but at some places it is being used as constant and and some places as string. So would like to keep as a const for cmd package only. Will make this unexported. let  me know is this does not sound ok.